### PR TITLE
API-3209 local database skeleton

### PR DESCRIPTION
**Pull request by jblefkove-va**
https://vajira.max.gov/browse/API-3209

Based on https://github.com/department-of-veterans-affairs/lighthouse-facilities/tree/master/facilities-local-db

### CODE REVIEW

File: patient-generated-data-local-db/local-db.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```
Comment (apeirats-va) [inline patient-generated-data-local-db/local-db.sh:1]: I'd add a help/clean/start option here, similar to the synthetic records' `run-local.sh`

File: patient-generated-data-local-db/src/test/java/gov/va/api/health/patientgenerateddata/Populaterator.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    @NonNull String password;
+
+    @NonNull String database;
+
+    @Override
+    @SneakyThrows
+    public Optional<Connection> bootstrapConnection() {
+      String bootstrapUrl =
+          String.format("jdbc:sqlserver://%s:%s;user=%S;password=%s", host, port, user, password);
+      return Optional.of(DriverManager.getConnection(bootstrapUrl));
```
Comment (apeirats-va) [inline patient-generated-data-local-db/src/test/java/gov/va/api/health/patientgenerateddata/Populaterator.java:119]: Should this be calling the connection method below instead?

Comment (jblefkove-va) [reply patient-generated-data-local-db/src/test/java/gov/va/api/health/patientgenerateddata/Populaterator.java:119]: They are subtly different, the other method includes an extra parameter

File: patient-generated-data-local-db/local-db.sh (hunk 1 above)
Comment (jblefkove-va) [reply patient-generated-data-local-db/local-db.sh:1]: The use case for DQ synthetic records is more complicated. In particular, rebuilding that db is expensive enough that you don't necessarily want to do it every time. For PGD and facilities, rebuilding is cheap, so the script is simplified: always drop the current container and build a fresh one

File: patient-generated-data-local-db/src/test/java/gov/va/api/health/patientgenerateddata/Populaterator.java (hunk 2 above)
Comment (apeirats-va) [reply patient-generated-data-local-db/src/test/java/gov/va/api/health/patientgenerateddata/Populaterator.java:119]: Nvm, I see the difference now. This one's used for dropping/creating the database itself. The other is used for an existing db's CRUD.

**Review by apeirats-va — APPROVED** [review]

**Pull request merged**